TidyPlatesInternal/TidyPlatesCore.lua

```diff
@@ -1327,6 +1327,10 @@ function CoreEvents:PLAYER_LOGIN()
   end
 end
 
+function CoreEvents:ARENA_OPPONENT_UPDATE()
+  Addon:SetBaseNamePlateSize()
+end
+
```

Comment:
Why is this here? Not sure how often ARENA_OPPONENT_UPDATE fires, but - if I don't remeber wrongly - several times, but that might be inefficient. Hopefully, there is a batter way.